### ISSUE TITLE
Need dashboard idea

### ISSUE THREAD

**Issue by adigunsherif** [adigunsherif/Django-School-Management-System#17]
What do you think should be the admin frontpage content? Do you have a design suggestions?

**Comment by sanjaypradeep** [comment]
Design discussion would be much helpful to accomplish this product much better.

**Comment by adigunsherif** [comment]
Right. if you have anything in mind, feel free to let us know so we can work on this asap.

**Comment by oyerohabib** [comment]
@sanjaypradeep @adigunsherif  I have some ideas on the dashboard like making some charts for visual representation of the summary of the essential things in the school.

Thanks

**Comment by mukesh12z** [comment]
different view for different users. principal sees overall activities of school(% of people passed, growth of pupil, overall attendance). Teachers dashboard view can be their class (attendance, progress report). parents dashboard can be attendance, progress of their pupil over time.
